### PR TITLE
add 'term_light' colorscheme

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

File: autoload/airline/themes/term.vim [hunk 1]
```diff
@@ -32,7 +32,7 @@ let s:R4 = [ '#E55345' , 9 ]
 let s:PA = [ '#94E42C' , 6 ]
 
 " Info modified
-let s:IM = [ '#40403C' , 238 ]
+let s:IM = [ '#40403C' , 7 ]
```
Comment (chrisbra) [inline autoload/airline/themes/term.vim]: why was this changed?

Comment (actionless) [reply autoload/airline/themes/term.vim]: i forgot to set that color when was submitting that colorscheme

**Comment by chrisbra** [comment]
alright, thanks.
